### ISSUE TITLE
3rd year check complete

### ISSUE THREAD

**Issue by lizbittner** [lonsun/richmond_trees-admin#8]
When I generate a report for tree care for October, the report includes trees that have already received their 3rd year check.  Is there some way to keep these off of the list?  Maybe I should use different search criteria?

**Comment by lonsun** [comment]
What is the exact criteria for no longer maintaining a tree?  Is it that it has had at least 3 checks?  Was checked in the last year and was planted more than 3 years ago?  If there is specific criteria I can automate leaving them off the report.

Alternatively, I can add a checkbox or that indicates that the tree is "no longer maintained" or something to that effect.

**Comment by lizbittner** [comment]
The criteria is that it was planted three years ago (or at least 30 months
ago) and has been checked at least three times.  (There are a few trees
that were checked twice in one year, so may technically have been checked 3
times, but should still be checked again during their 3rd year in the
ground.)  A checkbox may work well, something to the effect of "received
initial 3 checks" or "no longer maintained".  We may want to include them
in a search in the future, hopefully we can do another pruning cylce at 7
years in the ground.  Could we just do the search to include plants with
that check box?

**Comment by lonsun** [comment]
Ok...I will add the checkbox "received initial 3 checks" and automatically update the existing plantings based on that criteria.  You will be able to search using that checkbox value.

**Comment by lizbittner** [comment]
Thanks!

**Comment by lonsun** [comment]
This is done.  I've added an "initial_checks_received" flag to  plantings.  It is only accessible when editing a planting (as opposed to creating one).  

I also updated the existing data so that any tree that was planted more than 3 years ago, had at least 3 maintenance records, and was checked at least once in the last year, was updated with initial_checks_received = true.  A total of 101 records are now marked as having received the initial checks.
